action.yml

```diff
@@ -61,7 +65,7 @@ runs:
           --build-arg GITHUB_SHA="$GITHUB_SHA" \
           --build-arg GITHUB_REF="$GITHUB_REF" \
           ${{ inputs.build-options }} \
-          .
+          ${{ input.build-path }}

```

Comment:
```suggestion
          ${{ inputs.build-path }}
```

Reply:
It seems like _working-directory_  doesn't work with actions.  Found this
`An action is reusable/isolated code. Therefore there is no working directory for a step that uses an action (because an action can be pretty much anything). In regards to a custom action, you probably have to give the action an argument and pass a path` @dimw

Reply:
Done